### PR TITLE
Added status to DdiProviderRegistration entity

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 

#### Description
Added registration status of DdiProviderRegistration entity with a custom endpoint.

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Provider/Domain/Model/DdiProviderRegistration/DdiProviderRegistrationDto.php [hunk 1]
```diff
@@ -25,6 +27,27 @@ public static function getPropertyMap(string $context = '', string $role = null)
     {
         $response =  parent::getPropertyMap(...func_get_args());
 
+        if ($context === self::CONTEXT_STATUS) {
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DdiProviderRegistration/DdiProviderRegistrationDto.php:30]: add API tests

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**